sunpy/net/jsoc/attrs.py

```diff
@@ -71,6 +72,41 @@ def __init__(self, value):
         self.value = value
 
 
+class Cutout(DataAttr):
+    """
+    Select a cutout region to be processed by JSOC
+
+    Parameters
+    ----------
+    bottom_left
+    width
+    height
+    tracking
+    register
+    nan_off_limb
+    """
+
+    def __init__(self, bottom_left, width, height, tracking=False, register=False,
+                 nan_off_limb=False):
+        super().__init__()
+        self.value = {
+            't_ref': bottom_left.obstime.isot,
+            # JSOC input is disable tracking so take the negative
+            't': int(not tracking),
+            'r': int(register),
+            'c': int(nan_off_limb),
+            'locunits': 'arcsec',
+            'boxunits': 'arcsec',
+            'x': (bottom_left.Tx + width / 2).to('arcsec').value,
+            'y': (bottom_left.Ty + height / 2).to('arcsec').value,
+            'width': width.to('arcsec').value,
+            'height': height.to('arcsec').value,
+        }
+
+    def collides(self, other):
+        return False
```

Comment:
I *think*, this should collide with itself, i.e. you shouldn't have two of them in the same and block.

Reply:
Yep, that's correct. So this should just return `False` if `other` is an instance of `Cutout`?